Packs/SymantecBlueCoatProxySG/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,12 @@
+
+#### Modeling Rules
+
+##### Symantec BlueCoat ProxySG Modeling Rules
+
+- Updated the Symantec BlueCoat ProxySG Modeling Rules to align with improved field extraction in the Parsing Rule.
+
+#### Parsing Rules
+
+##### Symantec BlueCoat ProxySG Parsing Rules
+
+- Updated the Symantec BlueCoat ProxySG Parsing Rules to use the regexcapture function for extracting all fields from _raw_log.
```

Comment:
Updated the Symantec BlueCoat ProxySG Parsing Rule by using the **regexcapture** function to extract fields from the **_raw_log** field.